tesseract_python/tesseract_python/swig/tesseract/tesseract.i

```diff
@@ -0,0 +1,70 @@
+/**
+ * @file tesseract.i
+ * @brief SWIG interface file for tesseract/tesseract.h
+ *
+ * @author John Wason
+ * @date December 10, 2019
+ * @version TODO
+ * @bug No known bugs
+ *
+ * @copyright Copyright (c) 2019, Wason Technology, LLC
+ *
+ * @par License
+ * Software License Agreement (Apache License)
+ * @par
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * @par
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+%{
+#include <tesseract/tesseract.h>
+%}
+
+%shared_ptr(tesseract::Tesseract)
+
+namespace tesseract
+{
+class Tesseract
+{
+public:
+  using Ptr = std::shared_ptr<Tesseract>;
+  using ConstPtr = std::shared_ptr<const Tesseract>;
+
+  Tesseract();
+  virtual ~Tesseract() = default;
+
+  bool isInitialized() const;
+
+  bool init(tesseract_scene_graph::SceneGraph::Ptr scene_graph);
+  //bool init(tesseract_scene_graph::SceneGraph::Ptr scene_graph, tesseract_scene_graph::SRDFModel::ConstPtr srdf_model);
```

Comment:
Does this mean this method is not available in python? If not, what needs to change in Tesseract?

Reply:
I wasn't able to wrap the SRDF classes due to nested classes. It is also inflexible to adding and removing groups. I left the function in as a stopgap until something more polished was ready.

Reply:
Makes sense. This is one of the next classes on my list to rewrite.